### PR TITLE
Fix/fetch one section at a time

### PR DESCRIPTION
**Pull request by vektorustwo**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -42,20 +40,16 @@ export default class GuideHbg extends Component {
     LangService.loadStoredLanguage()
       .then(() => {
         // Check the network and load the content.
+        // TODO lang code needs to be moved into redux
```
Comment (bjornustwo) [inline index.js]: done!

File: index.js [hunk 2]
```diff
@@ -77,7 +71,7 @@ export default class GuideHbg extends Component {
   }
 
   static loadExistingDownloads() {
-    // TODO implement
+    // TODO remove dead code
```
Comment (bjornustwo) [inline index.js]: yep

File: src/data/repo/endpoints.js [hunk 3]
```diff
@@ -3,4 +3,4 @@
 export const _API_BASE = "https://api.helsingborg.se/event/json/wp/v2";
 export const _LANGUAGE_API_URL =
   "https://api.helsingborg.se/event/json/pll/v1/languages";
-export const API_BASE_URL = "https://hbg-guide.herokuapp.com";
+export const API_BASE_URL = __DEV__ ? "https://hbg-guide.herokuapp.com" : "https://guide-api.helsingborg.se/";
```
Comment (bjornustwo) [inline src/data/repo/endpoints.js]: 💥

**Comment by onnerfors** [comment]
jättebra jobbat @vektorustwo ! 💥

**Comment by vektorustwo** [comment]
Tack ska ni ha 😅

**Pull request merged**